src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -98,8 +97,6 @@ def __init__(self, **kwargs) -> None:
         self.__config = config
         self._reset_all_output_values()
         super().__init__(**config)
-        if (FEATURE_FLAGS.add_toolkit_output) and hasattr(self, "_append_tool_output") and self.add_tool_output:
-            self._append_tool_output()
```

Comment:
this means we're not enabling components as tools in this PR? Just getting rid of the feature flag?

Reply:
To enable a component as  tools, we need to add tool_mode as true for relevant inputs of the component and we shall get the ToolMode button in component.
<img width="302" alt="Screenshot 2024-11-07 at 5 11 31 PM" src="https://github.com/user-attachments/assets/b84ba475-7843-4c19-ac09-e0f02efc1d05">

PR to add Tool mode to relevant compoenents : #4439

Reply:
Gotcha thanks!